tests/api/admin-notes.php

```diff
@@ -491,6 +491,26 @@ public function test_delete_all_notes() {
 		$this->assertEquals( 4, count( $notes ) );
 	}
 
+	/**
+	 * Test deleting all the notes with a specific status.
+	 */
+	public function test_delete_all_notes_with_status() {
+		wp_set_current_user( $this->user );
+
+		// It deletes only unactioned notes.
+		$request = new WP_REST_Request( 'DELETE', $this->endpoint . '/delete/all' );
```

Comment:
Just a side comment... that's pretty bizarre that our API uses both the `DELETE` HTTP verb and a URL that includes `/delete`... such a mismash.